typescript/infra/helm/check-warp-deploy/templates/cron-job.yaml

```diff
@@ -17,7 +17,7 @@ spec:
           containers:
           - name: check-warp-deploy
             image: {{ .Values.image.repository }}:{{ .Values.image.tag }}
-            imagePullPolicy: IfNotPresent
+            imagePullPolicy: Always
```

Comment:
Not needed for this PR, but I wonder if it would be better to use the `latest`, I think it would always default `imagePullPolicy: Always` in that case. I think I tried latest and has issues

Reply:
I think the problem with latest is that images from branch CI will be available there